index.js

```diff
@@ -34,6 +34,10 @@ module.exports = {
         "no-useless-backreference": "error",
         "require-atomic-updates": "error",
         "eqeqeq": "error",
+        "semi": ["error", "always", { "omitLastInOneLineBlock": true, }],
```

Comment:
I would prefer to go in the opposite direction of **never** use semicolons, but if we are going to institute this rule it should be **always** use semicolons. We should not have exceptions like this because that just ends inconsistency, in my opinion. Inconsistency is what you are trying to fix here with this PR after all.

Reply:
I see your point, I'll make an update to it.